Core/Core/Inbox/ComposeMessage/View/ComposeMessageView.swift

```diff
@@ -47,7 +47,6 @@ public struct ComposeMessageView: View, ScreenViewTrackable {
 
     public var body: some View {
```

Comment:
Could you please reindent the whole body? Because of the removal of the `ScrollView {` and the removal of the BaseScreen's `}`, everything is off now.

Reply:
Could you please try again? select the whole file and autoindent it

Reply:
Fixed.